src/main/java/com/github/fishio/ICollisionArea.java

```diff
@@ -125,6 +145,14 @@ default boolean isReversed() {
 	double getSize();
 	
 	/**
+	 * Set the size of the collisionArea.
+	 * 
+	 * @param size
+	 * 		The new size.
+	 */
+	void setSize(double size);
```

Comment:
you can make this default method default using the increaseSize method.

```java
default void setSize(double size) {
     increaseSize(size - getSize());
} 
```

Or you could do it the other way around and make increaseSize(int size) default.

<organisation